.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -1,7 +1,7 @@
 <component name="InspectionProjectProfileManager">
   <settings>
-    <option name="PROJECT_PROFILE" value="Project Default" />
-    <option name="USE_PROJECT_PROFILE" value="true" />
+    <option name="PROJECT_PROFILE" value="vdmg" />
```

Comment:
Please rollback all the changes of .idea folder

Reply:
:-/ done: ```git checkout upstream/master -- .idea/```                                                                                                                                        

I need to add it into the global .gitignore